src/batch/src/executor/source.rs

```diff
@@ -49,7 +49,7 @@ pub struct SourceExecutor {
     schema: Schema,
     identity: String,
 
-    source_ctrl_opts: SourceCtrlOpts,
```

Comment:
so we don't support rate limit for batch source? I am not sure if it is expected.

Reply:
Currently yes we don't support. I'm not familiar with how the decision was made, maybe it was just not an urge, IDK.

Reply:
If this refers to DML, yes, it is currently unsupported.